examples/storage/cassandra/README.md

```diff
@@ -396,16 +396,13 @@ functionality, like a Deployment, ReplicaSet, Replication Controller, or Job.
 
 ## Step 4: Delete Cassandra StatefulSet
 
-There are some limitations with the Alpha release of PetSet in 1.3. From the [documentation](http://kubernetes.io/docs/user-guide/petset/):
-
-"Deleting the StatefulSet will not delete any pods. You will either have to manually scale it down to 0 pods first, or delete the pods yourself.
-Deleting and/or scaling a StatefulSet down will not delete the volumes associated with the StatefulSet. This is done to ensure safety first, your data is more valuable than an auto purge of all related StatefulSet resources. Deleting the Persistent Volume Claims will result in a deletion of the associated volumes."
```

Comment:
Deleting the Persistent Volume Claims will result in a deletion of the associated volumes *if they were dynamically provisioned*?